### PR TITLE
test: register and verifyEmail

### PR DESCRIPTION
**Pull request by graphtheoryiscool**
wooooo

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -38,7 +38,8 @@ export class APIClient {
 	*/
 
 	public register(data: RegisterData) {
-		return axios.post(`${this.apiBase}/register`, data, this.baseConfig);
+		const res = axios.post(`${this.apiBase}/register`, data, this.baseConfig);
+		return res;
```
Comment (amishshah) [inline src/index.ts]: Can you undo this change?

**Review by amishshah — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (graphtheoryiscool) [reply src/index.ts]: OH NO IM SORRY I FORGOT TO

**Review by amishshah — APPROVED** [review]
Wooo! LGTM! :rocket:
